core/templates/dev/head/pages/exploration_editor/EditorNavigationDirective.js

```diff
@@ -50,6 +50,8 @@ oppia.directive('editorNavigation', [
             }
           });
 
+          $scope.username = GLOBALS.username;
```

Comment:
Could this be: `$scope.userIsLoggedIn = Boolean(GLOBALS.username)`? That's all we care about here, really.